.eslintignore

```diff
@@ -0,0 +1,2 @@
+/public
+/bin
```

Comment:
Great 👍 
I guess you need to remove those files manually too in order to remove them from this PR. right ?

Reply:
yup.